spec/example_app/db/seeds.rb

```diff
@@ -10,12 +10,20 @@
 Order.destroy_all
 Customer.destroy_all
 Product.destroy_all
+Country.destroy_all
+
+Country.create(code: "US", name: "USA")
+Country.create(code: "CA", name: "Canada")
+Country.create(code: "CN", name: "China")
+Country.create(code: "RU", name: "Russia")
+Country.create(code: "AU", name: "Australia")
 
 100.times do
   name = "#{Faker::Name.first_name} #{Faker::Name.last_name}"
   Customer.create(
     name: name,
     email: Faker::Internet.safe_email(name),
+    country: Country.first,
```

Comment:
This iterates 100 times. Maybe do:

```ruby
country = Country.first
customers = Array.new(100) do
  {
    name: Faker::Name.name,
    email: Faker::Internet.safe_email(name),
    country: country
  }
end

Customer.create!(customers)
```

`Array.new` is faster than `Integer#times`; best practice to always use `Array.new`. Also, you don't have to call `Customer.create` 100 times, just once with an array for it to iterate over. We don't need this blazing fast. It's not inserting 2,000 records. However, we nearly made it do an extra 100 queries with `Customer.first` instead of just one. We could actually make it even simpler with taking the data returned from our `Country.create!` and sampling from there.